docs/sdk/authentication/session-sigs/intro.md

```diff
@@ -12,11 +12,12 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
 
 :::
 
-We refer to a session signature obtained from the user via session keys as a `SessionSig`.
+Session keys are a unique string of data generated during a specific online session between two or more entities. In our case, the keypair is a ed25519 keypair. It serves as a protective layer for identity verification by incorporating unique session-specific details such as date, time, and other contextual factors to generate a signature. A signature, known as a `SessionSig`, is obtained from the user through the session keys.
 
-`SessionSigs` are produced by a ed25519 keypair that is generated randomly on the browser and stored in local storage. The first step to producing `SessionSigs` is to first obtain an `AuthSig` through an authentication method like Google OAuth (example [here](https://github.com/LIT-Protocol/oauth-pkp-signup-example/blob/main/src/App.tsx#L398)). By specifying the session keypair's public key in the signature payload of the `AuthSig` - the `uri` field of the SIWE - users can choose which specific actions to delegate to the session keypair for operating upon certain resources.
+The Lit SDK uses the session keypair to sign all requests to the Lit Nodes. Each node in the Lit Network receives a unique signature for each request and can verify that the user owns the wallet address that signed the capability. The user’s `AuthSig` is sent along with the `SessionSig` as an attached "capability" to introduce capability to the `SessionSig`.
+
+Attaching capability is done by specifying the session keypair's public key as the `address` field of the `AuthSig` and the `capabilities` field. The `capabilities` field is an array of one or more signatures which authorize the `AuthSig` address (also our session keypair public key) to utilize the resources specified in the capabilities of the SIWE messages. These signatures contain the `AuthSig` `address` in their URI field, which can be seen in the example below.
```

Comment:
@awisniew207 how do you feel about rewriting the entire intro to:

In order to communicate with a Lit network, you must first authenticate yourself (i.e. prove who you are) with the network - this is done via Session Signatures.

Session Signatures are made using Session Keys which are generated on your behalf when you start a session by making a request to a Lit network using the Lit SDK. These Session Keys are a unique `ed25519` keypair generated by the Lit SDK locally, and are used to sign all requests to the Lit network for the current session.

When generating the Session Signatures, an Authentication Signature (Auth Sig) is used to prove who are to the Lit nodes, so that they can check if you're authorized to perform the action you're requesting to be done by the Lit network e.g. decrypting some data, signing a transaction with a PKP, transferring ownership of a PKP.

The Auth Sig is a [ERC-5573](https://eips.ethereum.org/EIPS/eip-5573) Sign-In with Ethereum Capabilities message that specifies a set of Lit resources (e.g. PKPs, Lit Actions) and specific [Lit Abilities](https://v6-api-doc-lit-js-sdk.vercel.app/enums/types_src.LitAbility.html) that the Session Keys are being granted usage of e.g. Signing transactions with a specific PKP that is owned by the address that produced the Auth Sig, execution of a specified Lit Action.

After making your request to the Lit network, each Lit node derives an identity from the Auth Sig and uses it to check if that identity owns/is authorized to use the Lit resource(s) and corresponding abilities your request is attempting to use.